pom.xml

```diff
@@ -29,16 +36,23 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <checkstyle.version>2.17</checkstyle.version>
         <checkstyle.methodNameFormat>^_?[a-z][a-zA-Z0-9]*$</checkstyle.methodNameFormat>
-        <jakarta.annotation-api.version>1.3.5</jakarta.annotation-api.version>
-        <jakarta.enterprise.cdi-api.version>2.0.2</jakarta.enterprise.cdi-api.version>
-        <jakarta.inject-api.version>1.0</jakarta.inject-api.version>
-        <jakarta.ws.rs-api.version>2.1.6</jakarta.ws.rs-api.version>
-        <microprofile-config-api.version>2.0</microprofile-config-api.version>
-        <microprofile-reactive-streams-operators.version>1.0.1</microprofile-reactive-streams-operators.version>
+        <version.plugin.checkstyle>3.1.1</version.plugin.checkstyle>
+
+        <!--  Dependencies - Jakarta  -->
+        <version.jakarta.api>9.0.0</version.jakarta.api>
```

Comment:
My apologies if I missed something, @andymc12, shouldn't it be 9.1.0? (https://jakarta.ee/specifications/platform/9.1/)

Reply:
Good catch @reta! And thanks for reviewing!!
I think I copied that from the parent pom, so I'll check to see if the parent pom needs to be updated.  I did a quick search, and this project doesn't use that property anywhere (it uses the individual Jakarta components instead), so I'll remove that line from this PR. Much appreciated!

Reply:
This version is not used anywhere. Please just remove it.